.github/workflows/ci.yml

```diff
@@ -7,6 +7,11 @@ jobs:
     runs-on: ubuntu-latest
     name: Ubuntu Bionic CI
     steps:
+      - name: Set env
+        run: |
+          export PATH=$PATH:/tmp/usd_binaries/bin
+          export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/tmp/usd_binaries/lib
+          export CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:/tmp/usd_binaries
```

Comment:
Fixed here https://github.com/ignitionrobotics/sdformat/pull/817/commits/92eca6331ff67586e00e86be81e2e9d9ce125258

PXR should not compile in bionic because of the cmake version